bib/french.bib

```diff
@@ -4946,8 +4946,8 @@ @BOOK(
 )
 
 @MVBOOK(
-	mercenne_1636,
-	author = {F. Marin Mercenne},
+	mersenne_1636,
```

Comment:
Will you then provide the commit with the changes required?
Please, note that added_on field should also be altered to reflect the last commit with the id change.